shims/spark35/src/main/scala/org/apache/gluten/sql/shims/spark35/Spark35Shims.scala

```diff
@@ -353,6 +354,12 @@ class Spark35Shims extends SparkShims {
   def getFileStatus(partition: PartitionDirectory): Seq[FileStatus] =
     partition.files.map(_.fileStatus)
 
+  def findRowIndexColumnIndexInSchema(sparkSchema: StructType): Boolean = false
```

Comment:
Why it's `false`?

Reply:
this function actually use to check if file `isSplitable`, i  will refactor and rename this function name as it confuses people.